### PR TITLE
Feature/csl1131 static peers

### PR DESCRIPTION
**Pull request by pva701**
* node/Main.hs is splitted on Param.hs, Secret.hs, Main.hs
* node/Main.hs is refactored
* wallet runners are moved to `Pos.Wallet.Web.Server.Full` (not the best place, but will be refactored in the future)
* added `--static-peers` flag and supported for node and full-wallet

### CODE REVIEW

File: src/Pos/Wallet/Web/Server/Full.hs [hunk 1]
```diff
@@ -1,31 +1,43 @@
...
+       , liftWMode
+       , runWStatsMode
+       , runWProductionMode
+       , runWStaticMode
+       , WalletStatsMode
+       , WalletProductionMode
+       , WalletStaticMode
        ) where
 
-import           Universum
```
Comment (gromakovsky) [inline src/Pos/Wallet/Web/Server/Full.hs]: Are you cereal?

Comment (pva701) [inline src/Pos/Wallet/Web/Server/Full.hs]: @gromakovsky che? che? do you mean "vegan"?

File: src/Pos/Wallet/Web/Server/Full.hs [hunk 2]
```diff
@@ -58,7 +74,82 @@ import           Pos.Wallet.Web.Server.Sockets (ConnectionsVar, getWalletWebSock
...
+    -> Production a
+runWStaticMode db conn =
+    runRawSBasedMode (runWalletWebDB db . runWalletWS conn . getNoStatsT) liftWMode
+
+liftWMode
+    :: ( Each '[MonadTrans] [t1, t2, t3]
+       , Each '[Monad] [m, t3 m, t2 (t3 m)]
+       )
+    => m a -> (t1 $ t2 $ t3 m) a
+liftWMode = lift . lift . lift
```
Comment (Martoon-00) [inline src/Pos/Wallet/Web/Server/Full.hs]: AFAIK we now have `powerLift`, could you remove it?

File: src/node/Main.hs [hunk 3]
```diff
@@ -18,345 +19,169 @@ import           Network.Transport.Abstract (Transport, hoistTransport)
...
+        => Transport (t0 $ t1 (RawRealMode ssc))
+    transportR = hoistTransport (lift . lift) transport
+
+    transportW ::
+        forall ssc t0 t1 t2 t3 .
+        ( Each '[MonadTrans] [t0, t1, t2, t3]
+        , Each '[Monad] [ t3 (RawRealMode ssc)
+                        , t2 (t3 (RawRealMode ssc))
+                        , t1 (t2 (t3 (RawRealMode ssc)))
+                        , t0 (t1 (t2 (t3 (RawRealMode ssc))))]
```
Comment (Martoon-00) [inline src/node/Main.hs]: :flushed:

File: src/node/Main.hs [hunk 4]
```diff
@@ -18,345 +19,169 @@ import           Network.Transport.Abstract (Transport, hoistTransport)
...
-                let allPlugins :: ([WorkerSpec (ProductionMode SscNistBeacon)], OutSpecs)
-                    allPlugins = mconcat [ wDhtWorkersProd
-                                         , convPlugins currentPlugins
-                                         , utwProd ]
-                runNodeProduction @SscNistBeacon
-                    (CLI.peerId commonArgs)
-                    (hoistTransport (lift . lift) transport)
-                    kademliaInst
-                    allPlugins
-                    currentParams ()
```
Comment (Martoon-00) [inline src/node/Main.hs]: Live's getting better :relaxed:

File: src/Pos/Wallet/Web/Server/Full.hs (hunk 1 above)
Comment (gromakovsky) [inline src/Pos/Wallet/Web/Server/Full.hs]: No, I meant `serious`. Why did you move it? https://github.com/serokell/serokell-util/blob/master/serokell-style.md#imports

Comment (Martoon-00) [inline src/Pos/Wallet/Web/Server/Full.hs]: Wee, when someone once said we have to place `Universum` separatelly, I thought is was a joke. Till now I placed it exactly like here

**Comment by pva701** [comment]
@gromakovsky hz. I tried but failed due to overlapped instances
